frontend/src/components/App.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { render, RenderResult } from '@testing-library/react';
+import { describe } from '@jest/globals';
+import App from './App';
+import { Axios } from 'axios';
+
+let documentBody: RenderResult;
+
+describe('<App />', () => {
+  beforeEach(async () => {
+    await process.nextTick(() => {});
+
+    documentBody = render(
+      <App/>
+    );
+
+    jest.spyOn(Axios.prototype, 'get').mockResolvedValue([]);
+  });
+
+  it('renders without crashing', () => {
+    expect(documentBody).toBeTruthy();
+  });
+
+  it('matches snapshot', () => {
+    const { baseElement } = documentBody;
+
+    expect(baseElement).toMatchSnapshot();
```

Comment:
I've never used Jest's snapshot testing - though have used tools like Percy for the same thing. This is definitely a useful alternative I'll keep in my back pocket!

Reply:
^^ This is a misunderstanding on my part. These are definitely not the same thing!

Reply:
Yeah snapshots are nice for that.